x-pack/test/functional/apps/index_management/feature_controls/index_management_security.ts

```diff
@@ -63,7 +63,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
         // The index_management_user has been given permissions to advanced settings for Stack Management Tests.
         // https://github.com/elastic/kibana/pull/113078/
-        expect(sections).to.have.length(2);
+        expect(sections).to.have.length(3);
```

Comment:
do we know why this had to change?

Reply:
Sure, this test checks how many sections show up in the navigation bar under Stack Management with the given user permissions (before this change, "Data" and "Kibana" were expected). Now that we made the Watcher privileges less strict and we don't require read access to the `.watches` index anymore, the permissions in this test allow Watcher to show up in the navigation bar as well, so now there is an additional section - "Alerts and Insights" which contains Watcher, which makes 3 sections in total in the navigation bar. Because of this, the test was failing in previous builds (see [these logs](https://s3.amazonaws.com/buildkiteartifacts.com/e0f3970e-3a75-4621-919f-e6c773e2bb12/0fda5127-f57f-42fb-8e5a-146b3d535916/0187387c-83c1-4a3d-8dc8-a15028ad04fe/0187387e-63bb-413d-9c7d-750b854bb8eb/target/test_failures/0187387e-63bb-413d-9c7d-750b854bb8eb_b708c83737c5dbd081b61d074d563871.html?response-content-type=text%2Fhtml&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQPCP3C7L6TSLD5VO%2F20230411%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20230411T085658Z&X-Amz-Expires=600&X-Amz-Security-Token=IQoJb3JpZ2luX2VjELb%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJHMEUCIQCQpuB%2BCOAOhCmSOOw5AcVYwCRPMCGoj1y%2BdwKj%2Be5cSgIgW6j04qjRAnQAFZKkVGbFVMmQGEIXrLkqdWIRusSt3tYq%2BgMIn%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FARAAGgwwMzIzNzk3MDUzMDMiDO9rRbSEr%2BWBpZ8suyrOA6Gxg9ykWGQceEzcbIFSINcA6%2Bu86McLbTAufjv3liknJPA%2BZumDjupeu9jtpqObzQ1sLRqRXhWbb0WPMwx0OiSHoHC%2BTVIWo%2FT%2F1qXfSPwZOMYfJn%2BZ%2F0yilDlNJg70pzaQRJATDPcNmr7K9r5t2WoXvtiwsB4dAKNIVQlWhV4z18vnEP7dUigJwho4tRgjZd%2F%2F%2F9%2FacxUmC%2FcGT5GTeqQI7cp1x68Xm9%2FHgeUS3HMbyC2Qfwb75cPjlNR6ird8TeXQwN4367LS8JEn20J%2FMD0QV2%2BgpHCKLMNwSWo%2BxLiQDjxCaOD8zIzJe9G%2FT2cNGexSM5O%2FF%2Fn8A1A%2FE%2FaTux8YimT6yGPFn62f9bgQFJsCBs0XNijRc4oC3fOy3Xe6fA8e1Dgh%2BN3bcDyK6vDjjCyBxBeb9nub44K4RsUsJzzrxIo6UnKfAQb729w5XsJl5YEl7vzZT0tIhU3aHQnFVIDI5L8tErTAvDXO8bRHBWMYwmwXJCcHgWXmjo1zCPklJJlZ%2FquKzxCcmvRVqszibh97Pq5OaIKuCJ37d69OvQHca8zXOH7y%2BK5TAqfOaZAA90sJe9hK6Yx450P6tMDwtTtb5gFqMmmAFe4Kuk%2BBkTCb69OhBjqlAQEMcT4joMOyb2qGiMwI242aE53o%2B6QJLIpN90fG8kmWDKjDprTAphyTa0guynC732CoRwRA%2Bxi1f3dQgxfdPWQeGCAz0YnVjAU7%2FGITBdaUXSSrz3o%2F9SZZ15x7NAoJLzD8wsTv6igb4bRwpy6W6rfiYRibzZ0%2FOLPN4MPkDnGQbaNZlDH8V%2BWW8v0vQ%2FSx03nrNyUiyJy5EFTqgVyMm%2BiuTPJ%2BrA%3D%3D&X-Amz-SignedHeaders=host&X-Amz-Signature=4d8cab6fa5de909d0d80d0b78ad87f01350a9578752e11fd4a07337482ff2483) for example), so it had to be modified to reflect the new scenario.

Reply:
Sure, this might be helpful for future maintenance.

Reply:
@ElenaStoeva Have you tested that watcher can indeed be used by a user without access to that index?

Reply:
@yuliacech no, I haven't. Would this be possible by creating a user in the "Users" plugin in Stack Management? If so, the test sets roles `global_dashboard_read` and `index_management_user` but I don't see these listed in the Roles dropdown list in the Users plugin so I wonder how we might give a user the same privileges as in the test.

Reply:
@ElenaStoeva Yes, you can create a user and a role to test this change. I think the best would be to create something similar to the role "watcher_admin" and change the permissions for indices.

Reply:
Added a comment to this test and verified that Watcher can be used by a user without access to the `.watches` index.